xml/System.Diagnostics.Tracing/EventWrittenEventArgs.xml

```diff
@@ -29,7 +29,33 @@
   <Interfaces />
   <Docs>
     <summary>Provides data for the <see cref="M:System.Diagnostics.Tracing.EventListener.OnEventWritten(System.Diagnostics.Tracing.EventWrittenEventArgs)" /> callback.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+
+Whenever an event is dispatched to an <xref:System.Diagnostics.Tracing.EventListener>, the <xref:System.Diagnostics.Tracing.EventListener.OnEventWritten&2A?displayProperty=nameWithType> callback method is invoked. It is passed an `EventWrittenEventArgs` instance that contains information associated with the event. 
+
+The following sections contain additional information about individual `EventWrittenEventArgs` properties.
+
+### ActivityId property
+
+Retrieving the value of the `ActivityId` property accesses thread local storage. This has the following implications:  
+  
+-   The `ActivityId` property is valid only for the duration of the <xref:System.Diagnostics.Tracing.EventListener.OnEventWritten%2A?displayProperty=nameWithType> callback.  
+  
+-   If the activity ID is changed during the callback, the activity ID returned by the `ActivityId` property is invalid.  
+
+### OSThreadId and TimeStamp properties 
+
+Starting with .NET Core 2.2, <xref:System.Diagnostics.Tracing.EventListener> objects can subscribe to native runtime events (such as GC, JIT, and threadpool events) in addition to events emitted by <xref:System.Diagnostics.Tracing.EventSource> objects. In previous versions of .NET Core and all versions of .NET Framework, the thead ID and timestamp can be gathered from the environment, because they are dispatched synchronously on the same thread that emitted them. Not all native runtime events can be dispatched synchronously, however. Some events, such as GC events, are emitted when managed thread execution is suspended. These events are buffered in native code and are dispatched by a dispatcher thread once managed code can execute again. Because these events are buffered, the environment cannot be used to reliably retrieve the thread ID and timestamp. Because of this, starting with .NET Core 2.2, thread ID and timestamp information are available as members of the `EventWrittenEventArgs` class.
+
+### RelatedActivityId property
```

Comment:
I would remove this section on RelatedActivityId.   I have put the important part into the remarks for the property.  I don't think it is importnat to indicate that one activity may have more than one RelatedActivityId (although it is true).    For what it is worth, a particular event can only have at most one RelatedActivityId.

Reply:
Remarks for an individual property aren't rendering, @vancem, which is why I've added detail on individual properties to the general remarks section (a build design bug). I'll incorporate the remarks you've added in the general remarks section.